src/core/server/base.js

```diff
@@ -125,6 +125,7 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
           includeSri: isDeployed,
           sriData,
           store,
+          trackingEnabled: config.get('trackingEnabled'),
```

Comment:
This is the setting to control if you want tracking to be enabled or not. 

If the id is missing it's internal enabled state will still be false even if trackingEnabled is set to true. Because trying to do stats without the id makes no sense.

Those are two separate things.